src/main/scala/uk/gov/nationalarchives/consignmentexport/Main.scala

```diff
@@ -73,8 +73,9 @@ object Main extends CommandIOApp("tdr-consignment-export", "Exports tdr files in
           tarPath = s"$basePath/$consignmentReference.tar.gz"
           _ <- bashCommands.runCommand(s"tar --sort=name --owner=root:0 --group=root:0 --mtime ${java.time.LocalDate.now.toString} -C $basePath -c ./${consignmentData.consignmentReference} | gzip -n > $tarPath")
           _ <- bashCommands.runCommand(s"sha256sum $tarPath > $tarPath.sha256")
-          _ <- s3Files.uploadFiles(config.s3.outputBucket, consignmentId, consignmentReference, tarPath)
-          _ <- graphQlApi.updateExportLocation(config, consignmentId, s"s3://${config.s3.outputBucket}/$consignmentReference.tar.gz", exportDatetime)
+          s3Bucket = if(consignmentData.consignmentType.contains("judgment")) { config.s3.outputBucketJudgment } else { config.s3.outputBucket}
+          _ <- s3Files.uploadFiles(s3Bucket, consignmentId, consignmentReference, tarPath)
+          _ <- graphQlApi.updateExportLocation(config, consignmentId, s"s3://${s3Bucket}/$consignmentReference.tar.gz", exportDatetime)
```

Comment:
Don't need the braces around the `$s3Bucket`: s"s3://$s3Bucket/$consignmentReference.tar.gz", exportDatetime)